src/ast/transform/AddNullariesToAtomlessAggregates.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+ * Souffle - A Datalog Compiler
+ * Copyright (c) 2020, Souffle Developers. All rights reserved
+ * Licensed under the Universal Permissive License v 1.0 as shown at:
+ * - https://opensource.org/licenses/UPL
+ * - <souffle root>/licenses/SOUFFLE-UPL.txt
+ */
+
+/************************************************************************
+ *
+ * @file AddNullariesToAtomlessAggregates.cpp
+ *
+ ***********************************************************************/
+
+#include "ast/transform/AddNullariesToAtomlessAggregates.h"
+#include "AggregateOp.h"
+#include "RamTypes.h"
+#include "ast/Aggregator.h"
+#include "ast/Argument.h"
+#include "ast/Atom.h"
+#include "ast/Attribute.h"
+#include "ast/Clause.h"
+#include "ast/LambdaNodeMapper.h"
+#include "ast/Literal.h"
+#include "ast/Node.h"
+#include "ast/Program.h"
+#include "ast/analysis/QualifiedName.h"
+#include "ast/analysis/Relation.h"
+#include "ast/analysis/TranslationUnit.h"
```

Comment:
```suggestion
#include "ast/QualifiedName.h"
#include "ast/Relation.h"
#include "ast/TranslationUnit.h"
```

I should have been more specific. The context github provided made the comment a tad confusing. Only TypeSystem has moved.